nucliadb/nucliadb/ingest/chitchat.py

```diff
@@ -135,3 +128,23 @@ async def socket_reader(
     async def close(self):
         self.chitchat_update_srv.close()
         self.task.cancel()
+
+
+def build_member_from_json(json):
```

Comment:
json is a bad name, please use a semantic variable

Reply:
@bloodbare Do you have any suggestion in mind ? From my point of view, `json` is ok so far since we cannot assert anything else about it before validating and perform the actual conversion. Something like `maybe_member_json_repr` can be used here but it adds just noise IMO.

Reply:
`member_serial` can be a good candidate BTW

Reply:
member serial is a good name. json its a bad name because its a common libeary and is not semantic